crates/ruff_cli/src/printer.rs

```diff
@@ -691,13 +731,29 @@ fn print_grouped_message<T: Write>(
     autofix_level: fix::FixMode,
     row_length: usize,
     column_length: usize,
+    jupyter_index: Option<&JupyterIndex>,
 ) -> Result<()> {
+    // Check if we're working on a jupyter notebook and translate positions with cell accordingly
+    let pos_label = if let Some(jupyter_index) = jupyter_index {
+        format!(
+            "cell {cell}{sep}{row}{sep}{col}",
```

Comment:
How does nbqa format / convey these?

Reply:
```
$ nbqa ruff /home/konsti/ruff/crates/ruff/resources/test/fixtures/jupyter/
crates/ruff/resources/test/fixtures/jupyter/valid.ipynb:cell_1:2:5: F841 [*] Local variable `x` is assigned to but never used
Found 1 error.
[*] 1 potentially fixable with the --fix option.

nbQA failed to process /home/konsti/ruff/crates/ruff/resources/test/fixtures/jupyter/not_json.ipynb with exception "JSONDecodeError('Expecting value: line 1 column 1 (char 0)')"
nbQA failed to process /home/konsti/ruff/crates/ruff/resources/test/fixtures/jupyter/invalid_extension.ipynb with exception "JSONDecodeError('Expecting value: line 1 column 1 (char 0)')"
nbQA failed to process /home/konsti/ruff/crates/ruff/resources/test/fixtures/jupyter/wrong_schema.ipynb with exception "KeyError('cells')"

If you believe the notebook(s) to be valid, please report a bug at https://github.com/nbQA-dev/nbQA/issues 
```

Underscore instead of space but otherwise same format

Reply:
Should we use an underscore? Wdyt? It might play better with regex parsers that rely on the structure of this output.